Weaver/WeaverTests/README.md

```diff
@@ -0,0 +1,20 @@
+About WeaverTests
+=================
+
+The WeaverTests are not included in the standard runner apps, unlike most of the adjacent tests.
+
+Instead, they are run within a unit test runner, in Xamarin Studio just with 
```

Comment:
_Inside a unit test runner_ as a phrase covers running inside VS inside one of multiple runners, as well as command line.